cypress/integration/editorCanPublishArticles.feature.js

```diff
@@ -1,4 +1,6 @@
-describe("editor can publish articles", () => {
+import { Button } from "semantic-ui-react";
```

Comment:
How is this used?

Reply:
haha, that got auto-imported when I forgot to add quotations apparently...